scss/base/_layout.scss

```diff
@@ -5,7 +5,7 @@
   display: none;
 
   @include respond-to('medium') {
-    @include absolute(top 10px left 10px);
+    @include absolute(top 8px left 20px);
```

Comment:
That being said, the mixin is crap. I mean, it's clever but this is not how I deal with position anymore. I'll change it to something like this:

```scss
@mixin position($position, $top: null, $right: null, $bottom: null, $left: null) {
  position: $position;
  top: $top;
  right: $right;
  bottom: $bottom;
  left: $left;
}

@mixin absolute($args...) {
  @include position(absolute, $args...);
}
```

```scss
.foo { 
  @include absolute($top: 8px, $left: 20px);
}
```